registrations/tasks.py

```diff
@@ -130,7 +130,18 @@ def get_messageset_schedule_sequence(short_name, weeks, voice_days,
     # determine how many times a week messages are sent e.g. 2 for '1,3'
     msgs_per_week = len(days_of_week.split(','))
     # determine starting message
-    next_sequence_number = msgs_per_week * weeks
+    # check if in prebirth stage - only starting messaging in week 10
+    if 'prebirth' in short_name:
+        next_sequence_number = msgs_per_week * (
+            weeks - settings.PREBIRTH_MIN_WEEKS)
+        if next_sequence_number <= 0:
+            next_sequence_number = 1
+    elif '13_52' in short_name:
+        next_sequence_number = msgs_per_week * (weeks - 13)
+        if next_sequence_number <= 0:
```

Comment:
make this `== 0` please. Rounding issues are fine, but if it goes to minus values something else has gone wrong and we want an exception in place.